source/catalog.html

```diff
@@ -7,25 +7,26 @@
   <body class="page__body">
 
     <header class="header">
-      <a class="header__logo-link">
-        <img src="img/logo-mobile.svg" width="33" height="37" alt="Cat energy">
-        <img class="logo-title" src="img/logo-title.svg" width="101" height="18" alt="Cat energy title">
+      <a href="index.html" class="header__logo-link">
+        <img class="logo-icon" src="img/logo-mobile.svg" width="33" height="37"
+          alt="Cat energy" aria-hidden="true">
+        <img class="logo-text" src="img/logo-title.svg" width="101" height="18" alt="Cat energy">
```

Comment:
Не, ну если так, то можно сделать отдельный блок `logo`, а то получается у нас сразу 2 блока создается.
```suggestion
      <a href="index.html" class="header__logo logo">
        <img class="logo__icon" src="img/logo-mobile.svg" width="33" height="37"
          alt="Cat energy" aria-hidden="true">
        <img class="logo__text" src="img/logo-title.svg" width="101" height="18" alt="Cat energy">
```